### PR TITLE
Style/run lint

### PR DESCRIPTION
**Pull request by fanny**
# Description

Run lint in all project

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Run `npm run lint` and see only warnings in output, and run some queries in playground, for see if functionalities works as expected.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/auth/route.js [hunk 1]
```diff
@@ -56,9 +43,9 @@ const requestAccessToken = (req, res) => {
   request.post(requestOptions, (error, response, body) => {
     let url;
     if (!error && response.statusCode === 200) {
-      const { access_token } = body;
-      
-      url = getClientURL(`${config.client.successPath}/${access_token}`);
+      const { access_token: acessToken } = body;
```
Comment (jadsonluan) [inline src/auth/route.js]: Variable name misspelled. 
**Current:** acessToken.
**Correct:** ac**c**essToken

Comment (jadsonluan) [inline src/auth/route.js]: Variable name misspelled.
**Correct**:  accessToken
**Current**: acessToken

Comment (jadsonluan) [reply src/auth/route.js]: Don't forget to change in the next line too.

File: package.json [hunk 2]
```diff
@@ -4,10 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest '.spec.js'",
+    "test": "jest 'contributors.spec.js'",
```
Comment (LukeHxH) [inline package.json]:
```suggestion
    "test": "jest '.spec.js'",
```

Run all tests.

**Pull request merged**